protein_lm/modeling/__init__.py

```diff
@@ -0,0 +1 @@
+from models import *
```

Comment:
```suggestion
from .models import *
```
I think you need these to be relative imports? They didn't work as is for me. 
Alternatively, instead of changing all of these to relative imports we can remove these lines and import them by specifying the full module paths in `test_coord_check.py`